3_visualize/src/plot_radial.R

```diff
@@ -0,0 +1,37 @@
+plot_radial_chart <- function(major_drought_periods, drought_events, color_scheme){
+  
+  # Using the major droughts, create concentric circles that are grey
+  ggplot(data = major_drought_periods, aes(x = angle, y = start))+
+    geom_ribbon(aes(ymin = start, ymax = end,
+                    xmin = angle, xmax = angle,
+                    group = name),
+                fill = color_scheme$drought_period_shading, alpha = 0.7) +
+    geom_violin(data = drought_events,
+                aes(x = CASC_angle, y = date, group = CASC),
+                color = NA, 
+                fill = color_scheme$drought_event_highlight,
+                scale = "count") +
+    scale_y_date(breaks = scales::date_breaks(width = '10 years'),
+                 #labels = NULL,
+                 labels = scales::date_format('%Y'), # use to check line-up on website
+                 limits = c(as.Date("1900-01-01"), 
```

Comment:
```suggestion
                 limits = c(as.Date("1900-01-01"),  # Start y axis 2 decades before data start so that violins don't all start at exact center of radial chart
```